src/consumer.rs

```diff
@@ -37,6 +37,45 @@ impl<
     }
 }
 
+/// Continuously consumes message from a Queue
+///
+/// A consumer represents the [basic.consume](https://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.consume) AMQP command.
+/// It continuously receives messages from the queue, as opposed to the
+/// [basic.get](https://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.get) command, which
+/// retrieves only a single message.
+///
+/// A consumer is obtained by calling [`Channel::basic_consume`] with the queue name.
+/// New messages from this consumer can be accessed by obtaining the iterator from the consumer.
+/// This iterator returns new messages in the form of a
+/// [`Delivery`] for as long as the consumer is subscribed to the queue.
+///
+/// It is important to acknowledge each message if acknowledgments are not disabled by calling
+/// [`Channel::basic_ack`] with the delivery tag of the message.
```

Comment:
ack is not the only option, there is also nack and reject

Reply:
Updated and linked to Delivery for further information.